Automattic/wp-calypso
# Command Palette: Remove GitHub deployments command single site sticker check.

# We need to wait until March 18th (GitHub deployments feature release date) before merging

Related to:
- https://github.com/Automattic/dotcom-forge/issues/6074

## Proposed Changes

* Removed conditionally checking if the GitHub deployments command should be added to the single site command list.

## Testing Instructions



* Upload the command palette app to your sandbox.
* Sandbox widgets.wp.com
* Open your browser's console and set the `isGHDeployAvailable` property to false before opening the command palette.
* Open the command palette and search for `GitHub`, you should see the GitHub deployments command on the command palette.

## Pre-merge Checklist
